Lesson 8: Literate Programming (#1994)
kframework/k: k-distribution/k-tutorial/1_basic/08_literate_programming/README.md

# Lesson 1.8: Literate Programming with Markdown

The purpose of this lesson is to teach a paradigm for performing literate
programming in K, and explain how this can be used to create K definitions
that are also documentation.

## Markdown and K

The K tutorial so far has been written in
[Markdown](https://guides.github.com/features/mastering-markdown/). Markdown, 

> for those not already familiar, is a lightweight plain-text format for styling

Reviewer: ```suggestion
s a lightweight plain-text format for styling
```

text. From this point onward, we assume you are familiar with Markdown and how
to write Markdown code. You can refer to the above link for a tutorial if you
are not already familiar.

> What you may not necessarily realize, however, is that the K tutorial is also

Reviewer: ```suggestion
However, the K tutorial is also
```
